src/workerd/api/pyodide/pyodide.h

```diff
@@ -0,0 +1,19 @@
+#pragma once
+
+#include <kj/common.h>
+#include <pyodide/pyodide.capnp.h>
+#include <workerd/api/pyodide/eval.h>
+#include <workerd/util/autogate.h>
+
+namespace workerd::api::pyodide {
+
+template <class Registry> void registerPyodideModules(Registry& registry, auto featureFlags) {
+  if (!util::Autogate::isEnabled(util::AutogateKey::BUILTIN_WASM_MODULES)) {
```

Comment:
Should we also gate this on the `--experimental` compat flag? at least for now.

Reply:
Sounds good to me/